### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by yeohcy7**
# Description

Not able to click Login button

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How to test?

- Please also list any relevant details for your test configuration
  * OS: Windows
  * Python version: 3.8
  * Browser: Chrome

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -23,7 +23,14 @@ def launch_browser():
 def log_in(browser, asnb_username, asnb_password):
 
     logging.info('🔑 Logging in')
-    browser.find_element_by_class_name("btn-login").click()
+#     browser.find_element_by_class_name("btn-login").click()
```
Comment (ngshiheng) [inline main.py]: Can you remove the commented line here?

File: main.py [hunk 2]
```diff
@@ -23,7 +23,14 @@ def launch_browser():
...
+    
+    aTagsInLi = browser.find_elements_by_css_selector('li a')
+    for a in aTagsInLi:
+        if "https://www.myasnb.com.my/uh/uhlogin/index?id=" in a.get_attribute('href'):
+            browser.get(a.get_attribute('href'))
+            break
+
```
Comment (ngshiheng) [inline main.py]: Can you resubmit a PR with passing flake 8?
```
./main.py:27:1: W293 blank line contains whitespace
./main.py:33:1: W293 blank line contains whitespace
✗ ERR - error while running flake8
```

Comment (ngshiheng) [reply main.py]: you can run these command (i configured these in the `lint.sh` file):
`flake8`
`autopep8 --in-place --aggressive -v main.py`

File: main.py (hunk 1 above)
Comment (yeohcy7) [reply main.py]: Try added more delay time but the btn-login element still not interactable. If you side tested ok, please close the pull request.

File: main.py (hunk 2 above)
Comment (yeohcy7) [reply main.py]: image
Done

**Pull request closed without merging**